docs/Bundler/bundlermethods.md

```diff
@@ -4,86 +4,134 @@ sidebar_position: 3
 custom_edit_url: https://github.com/bcnmy/docs/blob/master/docs/Bundler/integration.mdx
 ---
 
-# Bundler Methods
+# Methods
 
-Following are the methods that can be called on bundler instance
+## [estimateUserOpGas](https://bcnmy.github.io/biconomy-client-sdk/classes/Bundler.html#estimateUserOpGas)
+This method is used to estimate gas for the userOp. It returns estimates for PreVerificationGas, VerificationGas, and CallGasLimit for a given UserOperation. It requires passing a semi-valid/ dummy signature in userOp (e.g. a signature of the correct length and format). 
```

Comment:
Should be verificationGasLimit not VerificationGas
Can we also make it use camel casing for these variables?